icu4c/source/common/unicode/docmain.h

```diff
@@ -138,6 +138,11 @@
  *     <td>icu::DateFormat</td>
  *   </tr>
  *   <tr>
+ *     <td>Basic Time Zone, Rule Based Time Zone, Simple Time Zone, VTimeZone</td>
+ *     <td>basictz.h, rbtz.h simpletz.h, vtzone.h</td>
+ *     <td>icu::BasicTimeZone, icu::RuleBasedTimeZone, icu::SimpeTimeZone, icu::VTimeZone</td>
```

Comment:
We don't need to list the subclasses, but I am surprised that there is not already an entry for icu::TimeZone. I would either add that to "Calendars" (and rename to "Calendars & time zones") or make a row for just "time zones". There is no dedicated C API though; you can just repeat ucal.h for that if it's a separate row.

Reply:
Added to Calendars line.